ci/environment.yml

```diff
@@ -6,3 +6,7 @@ dependencies:
   - dask
   - matchpy
   - pyarrow
+  - pip
+  - pip:
+    - git+https://github.com/mrocklin/matchpy@play
```

Comment:
How do you want to deal with this long term? Are you hoping to contribute to matchpy?